src/Toml/Bi/Combinators.hs

```diff
@@ -272,7 +239,7 @@ table bi key = Bijection input output
         let newToml = execState (biWrite bi a) toml
         a <$ modify (\(TOML vals tables) -> TOML vals (Prefix.insert key newToml tables))
 
-    rethrowNotFound :: DecodeException -> Env a
-    rethrowNotFound (KeyNotFound name)   = throwError $ IncompleteTable key name
-    rethrowNotFound (TableNotFound name) = throwError $ IncompleteTable key name
-    rethrowNotFound e                    = throwError e
+    addTableName :: DecodeException -> Env a
+    addTableName (KeyNotFound name)   = throwError $ KeyNotFound (key <> name)
+    addTableName (TableNotFound name) = throwError $ TableNotFound (key <> name)
+    addTableName e                    = throwError e
```

Comment:
Probably it's better to call it `tableNameHandler` or something like this just to see that it's for error handling?

Reply:
Good catch! I also forgot about one error here...